### PR TITLE
docs: improve "Sending a Pull Request"

### PR DESCRIPTION
**Pull request by kenjis**
**Description**
- add git command examples

**Checklist:**
- [x] Securely signed commits
- [] Component(s) with PHPDoc blocks, only if necessary or adds value
- [] Unit testing, with >80% coverage
- [] User guide updated
- [] Conforms to style guide

### CODE REVIEW

File: contributing/pull_request.md [hunk 1]
```diff
@@ -214,18 +214,28 @@ The best way to contribute is to fork the CodeIgniter4 repository, and "clone" t
...
 4. [Clone](https://help.github.com/en/articles/cloning-a-repository) your CodeIgniter repository: `git@github.com:<your-name>/CodeIgniter4.git`
-5. Create a new [branch](https://help.github.com/en/articles/about-branches) in your project for each set of changes you want to make.
-6. Fix existing bugs on the [Issue tracker](https://github.com/codeigniter4/CodeIgniter4/issues) after confirming that no one else is working on them.
-7. [Commit](https://help.github.com/en/desktop/contributing-to-projects/committing-and-reviewing-changes-to-your-project) the changed files in your contribution branch.
-8. Commit messages are expected to be descriptive of what you changed specifically. Commit messages like
-   "Fixes #1234" would be asked by the reviewer to be revised.
+   - `> git clone git@github.com:<your-name>/CodeIgniter4.git`
+5. Install Composer dependencies.
+   - `> cd CodeIgniter4/`
+   - `> composer install`
```
Comment (paulbalandan) [inline contributing/pull_request.md:220]: Composer 2 suggests using `composer update` when installing without a lock file

File: contributing/pull_request.md [hunk 2]
```diff
@@ -214,18 +214,28 @@ The best way to contribute is to fork the CodeIgniter4 repository, and "clone" t
...
+6. Create a new [branch](https://help.github.com/en/articles/about-branches) in your project for each set of changes you want to make.
+   - `> git checkout -b <new-branch-name>`
+7. Fix existing bugs on the [Issue tracker](https://github.com/codeigniter4/CodeIgniter4/issues) after confirming that no one else is working on them.
+8. [Commit](https://help.github.com/en/desktop/contributing-to-projects/committing-and-reviewing-changes-to-your-project) the changed files in your contribution branch.
+   - `> git commit`
+   - Commit messages are expected to be descriptive of what you changed specifically. Commit messages like
+     "Fixes #1234" would be asked by the reviewer to be revised.
```
Comment (paulbalandan) [inline contributing/pull_request.md:227]: Can make this as one line

File: contributing/pull_request.md (hunk 1 above)
Comment (kenjis) [reply contributing/pull_request.md]: Fixed.

File: contributing/pull_request.md (hunk 2 above)
Comment (kenjis) [reply contributing/pull_request.md]: Fixed.

**Review by paulbalandan — APPROVED** [review]

**Pull request merged**